stable/metrics-server/templates/metrics-server-deployment.yaml

```diff
@@ -25,4 +25,8 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
             - /metrics-server
+{{- if .Values.kubeletReadOnlyPort }}
```

Comment:
This change conflicts with the more generic https://github.com/helm/charts/pull/6620 that was made to solve the same problem (but doesn't include the RBAC change)

Reply:
Was about to comment the same thing. In a sense, I prefer the approach of #6620 because this one now hard-codes a port, `insecure`  setting,...